trade-finance-manager-api/src/v1/controllers/utilisation-reports/get-selected-fee-records-details.controller.ts

```diff
@@ -0,0 +1,16 @@
+import { Request, Response } from 'express';
+import { AxiosError } from 'axios';
+import api from '../../api';
+
+export const getSelectedFeeRecordsDetails = async (req: Request<{ id: number }, object, { feeRecordIds: number[] }>, res: Response) => {
+  try {
+    const { id } = req.params;
+    const { feeRecordIds } = req.body;
+    const SelectedFeeRecordsDetails = await api.getSelectedFeeRecordsDetails(id, feeRecordIds);
+    res.status(200).send(SelectedFeeRecordsDetails);
+  } catch (error) {
+    const errorMessage = 'Failed to get selected fee records details';
+    console.error(errorMessage, error);
+    res.status((error as AxiosError).response?.status ?? 500).send(errorMessage);
```

Comment:
I'm not sure we can always assume this is an `AxiosError`, so this might want to be `error?.response?.status ?? 500` (it probably will be an axios error in prod, but when working on a new feature we may get a `TypeError` for example here which would cause the container to crash unexpectedly due to `response` not existing on `error` in this case, which might be quite hard to debug)

Reply:
Ah fairs, I did just blindly copy this 😬 `error?.response?.status ?? 500` sadly doesn't play nice but found another controller with a solution and copied that

Reply:
Nice solution :)